### PR TITLE
Render market list using warp sync data

### LINKED ISSUE #8220: Market List should Render for search engine bots
Moving warp loading code into SDK-lite so this can happen during hotload. If this doesn't work we'll go with plan b.

### PR DESCRIPTION
**Pull request by justinbarry**
This seems to work but is kinda ugly.

closes #8220 

- [x] Cache warp sync file

- [x] Pin UI on load

- [x] Hotload all markets.

### CODE REVIEW

File: packages/augur-ui/src/modules/markets-list/containers/markets-view-container.ts [hunk 1]
```diff
@@ -142,6 +145,11 @@ const mapDispatchToProps = (
         pageNumber
       )
     ),
+  hotLoadMarketList: async (cb) => {
+    const marketsInfo = await augurSdkLite.get().getMarketCreatedLogs('0x919274CE3Ba5a0D0f11071Bb9fb402AE5c23381B');
```
Comment (JohnDanz) [inline packages/augur-ui/src/modules/markets-list/containers/markets-view-container.ts]: how come you put a hardcoded address in here? testing?

Comment (justinbarry) [reply packages/augur-ui/src/modules/markets-list/containers/markets-view-container.ts]: Good catch! Fixed

**Comment by pgebheim** [comment]
Good point on PIN ui on load.

File: packages/augur-ui/src/modules/market/components/market-view/market-view.tsx [hunk 2]
```diff
@@ -237,9 +237,17 @@ export default class MarketView extends Component<
...
 
-    if (this.props.canHotload && prevProps.canHotload !== this.props.canHotload  && !tradingTutorial && marketId) {
+    if (
+      typeof this.props.marketId !== 'undefined' &&
+      typeof this.props.canHotload !== 'undefined' &&
+      !tradingTutorial &&
+      // We can hotload now.
+      (prevProps.canHotload === this.props.canHotload ||
+        // The market id changed.
+        prevProps.marketId !== this.props.marketId)
```
Comment (pgebheim) [inline packages/augur-ui/src/modules/market/components/market-view/market-view.tsx:247]: ah clever

**Review by pgebheim — APPROVED** [review]
Looks good wanting another set of eyes on react stuffs (@JohnDanz)

**Pull request merged**